import.py

```diff
@@ -583,6 +583,80 @@ def write_unified_cpp_file(dir):
       for cpp_file in cpps:
         f.write("#include \"{}\"\n".format(cpp_file))
 
+def extract_pref_from_line(line):
+    m = re.match(r'pref\((".*?"),\s*(.*)\);', line)
+    if not m:
+        print("extract_pref_from_line() failed on {}".format(line))
+        sys.exit(1)
```

Comment:
It would be a bit nicer to write errors on stderr and chain-up the exit code to main(). I'll propose those changes in a future PR.